components/brave_wallet_ui/common/wallet_api_proxy_observers.ts

```diff
@@ -192,14 +191,11 @@ export function makeJsonRpcServiceObserver (store: Store) {
 
   export function  makeBraveWalletPinServiceObserver (store: Store) {
     const braveWalletServiceObserverReceiver = new BraveWallet.BraveWalletPinServiceObserverReceiver({
-      onTokenStatusChanged: function (service, token, status) {
-        store.dispatch(WalletPageActions.updateNftPinningStatus({
-          token,
-          status
-        }))
+      onTokenStatusChanged: function (_service, _token, _status) {
+        store.dispatch(walletApi.endpoints.getNftsPinningStatus.initiate())
       },
       onLocalNodeStatusChanged: function (status) {
-        store.dispatch(WalletPageActions.updateLocalIpfsNodeStatus(status))
+        store.dispatch(walletApi.endpoints.updateLocalIpfsNodeStatus.initiate(status))
```

Comment:
Can we try this instead?
```suggestion
        store.dispatch(
          walletApi.util.invalidateTags(['LocalIpfsNodeStatus'])
        )
```